conda_forge_tick/make_migrators.py

```diff
@@ -206,6 +206,15 @@ def add_arch_migrate(migrators: MutableSequence[Migrator], gx: nx.DiGraph) -> No
             ),
         )
 
+    with fold_log_lines("making win-arm64 migrator"):
+        migrators.append(
+            WinArm64(
+                graph=total_graph,
+                pr_limit=PR_LIMIT,
+                name="arm64 win addition",
```

Comment:
No piggy back migrators?

Reply:
Im not sure what a piggy back migrator is?

Reply:
If you merge main into your pr, there is now a default set you can use with a function to instantiate them. See the other samples.

Reply:
Could we add piggy backs once we have the base packages migrated?

Reply:
Sure, but the first 20-50 packages are probably good anyways, right? I just want us to get started with bootstrapping win-arm64 :)

Reply:
No, it's not. I'd rather one person spend time with writing the migrator than 10 people all trying to figure out what went wrong.

Reply:
We can document what needs to be done in the tracking issue (or a separate one) that we link to in the opening comment of the bot. That way we avoid the "X people have to figure it out themselves" thing, even if some other modifications aren't yet automated.

I agree that we should get the migration started. IMO perfection is the enemy of the good here - we can improve as we scale up (and based on what we learn in the process).

Reply:
> We need to support windows with `UpdateCMakeArgsMigrator` and `GuardTestingMigrator`

Although those should probably be considered baseline.